scripts/zones/Nyzul_Isle/mobs/Cerberus.lua

```diff
@@ -13,11 +13,11 @@ entity.onMobSpawn = function(mob)
     mob:setMod(xi.mod.DOUBLE_ATTACK, 20)
     mob:setMod(xi.mod.REGEN, 10) -- validate
     -- mdt already set in mob family mods
-    mob:setMod(xi.mod.POISONRES, 100)
-    mob:setMod(xi.mod.PARALYZERES, 100)
-    mob:setMod(xi.mod.BLINDRES, 100)
-    mob:setMod(xi.mod.SILENCERES, 100)
-    mob:setMod(xi.mod.SLOWRES, 125)
+    mob:setMod(xi.mod.POISON_MEVA, 100)
+    mob:setMod(xi.mod.PARALYZE_MEVA, 100)
+    mob:setMod(xi.mod.BLIND_MEVA, 100)
+    mob:setMod(xi.mod.SILENCE_MEVA, 100)
+    mob:setMod(xi.mod.SLOW_MEVA, 125)
```

Comment:
We should move these to mob_pools.sql eventually

Reply:
And why slow_meva 125%?

Reply:
This is part of their new magic rewrite - we should not include any of this I believe